ebmlite/util.py

```diff
@@ -402,3 +405,34 @@ def pprint(el, values=True, out=sys.stdout, indent="  ", _depth=0):
                 out.write("\n")
 
     out.flush()
+
+
+#===============================================================================
+#
+#===============================================================================
+
+def printSchemata(paths=None, out=sys.stdout, absolute=True):
+    """ Display a list of schemata in `SCHEMA_PATH`. A thin wrapper for the
+        core `listSchemata()` function.
+
+        @param out: A file-like stream to which to write.
+    """
+    out = out or sys.stdout
+    newfile = isinstance(out, (str, pathlib.Path))
+    if newfile:
+        out = open(out, 'w')
+
+    try:
```

Comment:
not that it really makes a difference but I'd like the `open` function to be called within the try block to be safe.

Reply:
I *want* it to fail gracelessly here. The file won't be open if the `open()` failed, and even if it did, the file would close when leaving the scope of the function. It would complicate the `finally:`, too, since it exists only to close the file and `out` wouldn't be defined if the `open()` failed.

Reply:
yeah, that's fair.  I just end up getting worried about someone adding something in between down the road, but that's not really a realistic concern.